test/mongooseim_loglevel_SUITE.erl

```diff
@@ -0,0 +1,196 @@
+    ?assertEqual([], After -- Before);
+log_at_level(LName) ->
+    %% When current log level is L and we log on each possible level...
+    Before = get_log(?logfile),
+    [ logger:log(LevelName, "match-this ~s", [LevelName]) || {_, LevelName} <- levels(), LevelName /= none ],
+    %% ...then for each sensible level (i.e. less or equal to current level)
+    %% we get a line in the log file.
+    ExpectedContents = levels_less_than_or_equal_to(LName),
+    After = case get_at_least_n_log_lines(?logfile,
+                                          length(Before) + length(ExpectedContents),
+                                          timer:seconds(5)) of
+                timeout -> ct:fail("timed out waiting for messages to reach the log file");
+                Res -> Res
+            end,
+    LinesDiff = filter_out_non_matching(After -- Before, <<"match-this">>),
+    LinesWithExpectedContents = lists:zip(LinesDiff, ExpectedContents),
+    [ ?assert('contains?'(Line, ExpectedLevel))
+      || {Line, ExpectedLevel} <- LinesWithExpectedContents ].
+
+log_at_custom_level(_Config) ->
+    %% given logging on custom log level for the helper module
+    mongoose_logs:set_global_loglevel(critical),
+    mongoose_logs:set_module_loglevel(mongooseim_loglevel_SUITE_helper, debug),
+    %% when we log from here and from the helper module
+    Before = get_log(?logfile),
+    logger:log(debug, "suite", []), % <- This is not supposed to be logged
+    mongooseim_loglevel_SUITE_helper:log(debug, "helper module", []), % <- but this should
+    %% then
+    After = case get_at_least_n_log_lines(?logfile, length(Before) + 1, timer:seconds(5)) of
+                timeout -> ct:fail("timeout waiting for messages to reach the log file");
+                Res -> Res
+            end,
+    Diff = After -- Before,
+    %% ...nothing logged from the suite reaches the log file
+    ?assertEqual([], filter_out_non_matching(Diff, <<"suite">>)),
+    %% ...logs from the helper module are found in the log file
+    [LogLine] = filter_out_non_matching(Diff, <<"helper module">>),
+    ?assert('contains?'(LogLine, <<"debug">>)).
+
+%%
+%% Helpers
+%%
+
+levels() ->
+    [
+     {-1, none},
+     {0, emergency},
+     {1, alert},
+     {2, critical},
+     {3, error},
+     {4, warning},
+     {5, notice},
+     {6, info},
+     {7, debug}
+    ].
+
+mongoose_logger_running() ->
+    File = ?logfile,
+    HandlerID = disk_log,
+    HandlerModule = logger_disk_log_h,
+    HandlerConfig = #{config => #{
+                        file => "log/mongooseim.log",
+                        type => wrap,
+                        max_no_files => 5,
+                        max_no_bytes => 2097152
+                       }
+                     },
+    ok = logger:add_handler(HandlerID, HandlerModule, HandlerConfig),
+    FileBackend = {HandlerID, File},
+    {ok, FileBackend}.
+
+levels_less_than_or_equal_to(LName) ->
+    {L, LName} = lists:keyfind(LName, 2, levels()),
```

Comment:
`L` is a very unfortunate name for a variable, can you please replace with a word name or at least an abbreviation of some kind of something like that?